src/common/tools.ts

```diff
@@ -371,3 +371,6 @@ export const getLocalMessage = (path: string) => {
   const messages = i18n.global.messages[local]
   return get(messages, path)
 }
+
+export const getBridgeIconKey = (value: string) =>
+  value.indexOf('influxdb') > -1 ? `influxdb` : value
```

Comment:
Why the special handling of influxdb icon?

Reply:
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/26165221/183790324-26af0e82-ef0d-4e75-851f-289beb7e075e.png">

Reply:
well, there is some mistakes cause CI failed, I see see